include/maths/analytics/CBoostedTreeHyperparameters.h

```diff
@@ -65,6 +65,10 @@ class CBoostedTreeParameter final {
         : m_Value{value}, m_MinValue{value}, m_MaxValue{value}, m_LogSearch{logSearch == E_LogSearch} {
     }
 
+    CBoostedTreeParameter(const CBoostedTreeParameter<T>& other) = default;
+    CBoostedTreeParameter& operator=(const CBoostedTreeParameter<T>& other) = default;
+    ~CBoostedTreeParameter() = default;
```

Comment:
Was there a reason to add these?

Reply:
You are right, these default definitions are not necessary here.